lib/seattleflu/id3c/cli/command/etl/redcap.py

```diff
@@ -688,3 +688,16 @@ def build_questionnaire_items(question: str) -> Optional[dict]:
         return create_resource_entry(questionnaire_reseponse_resource, full_url)
 
     return None
+
+
+def extract_date_from_survey_timestamp(record: REDCapRecord, survey_name: str) -> Optional[str]:
+    """
+    Extracts as a string the date component of the *survey_name* REDCap survey timestamp, the system
+    timestamp that is captured automatically and is not dependent on the client. This timestamp
+    is in local (Pacific) time.
+    """
+    if record and survey_name and record.get(f'{survey_name}_timestamp'):
+            return datetime.strptime(record.get(f'{survey_name}_timestamp'),
+                '%Y-%m-%d %H:%M:%S').strftime('%Y-%m-%d')
+    else:
+        return None
```

Comment:
A couple of minor style suggestions.

The `return` statement on line 700 is over-indented.

Because we `return` in the `if` statement, we don't need to write an `else` statement. You can write, `return None` after the `if` statement is finished for the same effect.

Reply:
I fixed that.